src/In.ProjectEKA.DefaultHip/Resources/demoPatients.json

```diff
@@ -124,10 +143,12 @@
     "careContexts": [
       {
         "referenceNumber": "NCP1112",
+        "maskedReferenceNumber": "NCPXXXX",
         "display": "National Cancer program"
       },
       {
         "referenceNumber": "NCP11122",
+        "maskedReferenceNumber": "NCPXXXXX",
         "display": "National Cancer program -  Episode 2"
       }
     ]
```

Comment:
I would prefer to keep these masked in a different file to not pollute the patients.json.

Reply:
moved the mapping to maskedReferences.json and removed from demoPatients.json